src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -59,6 +59,11 @@ internal WebApplicationBuilder(WebApplicationOptions options, Action<IHostBuilde
 
         bootstrapHostBuilder.ConfigureWebHostDefaults(webHostBuilder =>
         {
+            webHostBuilder.ConfigureServices(services =>
+            {
+                services.AddRouting();
+            });
+
```

Comment:
This is already added.

```suggestion
```

Reply:
I made this change because I changed ```UseKestrel``` to call ```AddRoutingCore(...)``` instead of ```AddRouting(...)``` since we use this method in the slim builder scenario. I'm assuming it is the usage in ```UseKestrel(...)``` you are referring to when you say this is already added?

Reply:
Yep! Understood.